examples/future_campaign/multi_sweep.py

```diff
@@ -0,0 +1,347 @@
+                                                    Target_Age_Max=max_age
+                                                    )
+    campaign.add(one_time_campaign)
+    return {
+        "start_day": values['start_day_offset'],
+        'efficacy': values['efficacy'],
+        'coverage_camp': camp_coverage,
+        'decay': values['decay_constant']
+    }
+
+def sweep_config_func(config, values):
+    config.parameters.Typhoid_Acute_Infectiousness = values['typhoid_acute_infectiousness']
+    config.parameters.Typhoid_Exposure_Lambda = values['typhoid_exposure_lambda']
+    return {'Typhoid_Acute_Infectiousness': values['typhoid_acute_infectiousness'], 'Typhoid_Exposure_Lambda': values['typhoid_exposure_lambda']}
+
+def get_sweep_builders(camp_sweep_list, config_sweep_list, add_vax_fn=add_vax_intervention):
+    """
+    Build simulation builders.
+    Args:
+        kwargs: User inputs may overwrite the entries in the block.
+
+    Returns:
+        lis of Simulation builders
+    """
+    builder = SimulationBuilder()
+    funcs_list = [[
+        ItvFn(add_vax_fn, ce),
+        partial(set_param, param='Run_Number', value=x),
+        CfgFn(sweep_config_func, y)
+    ]
+        for ce in camp_sweep_list  # for sweep on sweep_list
+        for x in range(2)  # for sweep Run_Number
+        for y in config_sweep_list
+    ]
+
+    builder.add_sweep_definition(sweep_functions, funcs_list)
+
+    return [builder]
+
+
+def run( sweep_choice="All", age_targeted=True, binary_immunity=True ):
+    platform = Platform("SLURM", node_group="idm_48cores", priority="Highest") 
+    task = EMODTask.from_default2(config_path="config.json", eradication_path=manifest.eradication_path,
+                                  campaign_builder=build_camp, demog_builder=build_demog, schema_path=manifest.schema_file,
+                                  param_custom_cb=set_param_fn, ep4_custom_cb=None)
+    # normally we don't force-set parameters at this point
+    task.config.parameters.Demographics_Filenames = ["demographics.json","TestDemographics_pak_updated.json"]
+    task.config.parameters.Death_Rate_Dependence = "NONDISEASE_MORTALITY_BY_YEAR_AND_AGE_FOR_EACH_GENDER"
+    task.config.parameters.Birth_Rate_Dependence = "INDIVIDUAL_PREGNANCIES_BY_AGE_AND_YEAR"
+    # this is dumb
+    task.common_assets.add_directory(assets_directory=manifest.assets_input_dir)
+    task.set_sif(manifest.sif)
+
+    # Create simulation sweep with builder
+    def get_sweep_list_from_values(start_day_offset, vax_effs, cov, decay):
+        sweep_list = []
+        combinations = list(itertools.product(start_day_offset, vax_effs, cov, decay))
+        for c in combinations:
+            sweep_list.append({'start_day_offset': c[0], 'efficacy': c[1], 'coverage': c[2], 'decay_constant': c[3]})
+        return sweep_list
+
+    def get_sweep_list_full():
+        start_day_offset = [1]
+        vax_effs = np.linspace(0.1, 1.0, 10)
+        decay = [1, 365, 3650, 365000]
```

Comment:
decay = [1, 365, 3650, 365000]
last value throw out of range error. May change to 365*100